[MongoDB] Adiciona explicações (#1)
LucasSGomide/MongoDB: README.md

```
db.collection.deleteMany({ status : "A" });
```

Delete **all** documents that match a specified filter.

### EXISTS

Check if a field exists in the document.
  - Use true to find a field that exists.
  - Use false to find documents in which a field does not exist.

```
db.collection.find({ qty: { $exists: true } })
```

The query returns the documents that contain the field `qty`, including documents where the field value is `null`.

```
db.collection.find({ qty: { $exists: false } })
``` 

The query returns only the documents that do not contain the field `qty`.

### FIND

```
db.collection.find({});
``` 

Selects all documents in a collection.

```
db.collection.find({}, { name: 1 });
``` 

The second parameter (optional) specifies the fields to return in the document.

```
db.collection.find({ releaseYear: "2021" }, { title: 1, author: 1, _id: 0 });
``` 

Returns all documents that match the query, and display only the `title` and `author`.

### INSERT

```
db.createCollection( "minhaColecao4", { collation: { locale: "fr" } } );
```

Create a new collection.

```
db.collection.insertOne({ chave: "Valor", chave2: "Valor2" });
``` 

> Insert only one document in the collection. If the collection doesn't exist, mongodb automatically create it.

Reviewer: ```suggestion
Insert only one document in the collection. If the collection doesn't exist, mongodb automatically creates it.
```

Reviewer: Não tenho certeza, mas me soa correto kk

```
db.collection.insertMany([ { chave: "Valor", chave2: "Valor2" }, { chave: "Valor", chave2: "Valor2" } ]);
```

> Insert more than one documents in a collection.

Reviewer: ```suggestion
Insert more than one document in a collection.
```

### LIMIT

```
db.collection.find("<query>").limit("<número>");
```

Specify the number of documents to be returned.

```
db.collection.find().skip(2); 
```

> Specify the number of documents to be skipped. It control from where mongodb will begin to return results.

Reviewer: ```suggestion
Specify the number of documents to be skipped. It controls where mongodb will begin to return results.
```
